apps/aecore/src/aec_headers.erl

```diff
@@ -100,3 +109,24 @@ deserialize_from_binary(B) ->
 hash_header(H) ->
     {ok, BinaryH} = serialize_to_binary(H),
     {ok, aec_sha256:hash(BinaryH)}.
+
+serialize_pow_evidence(Ev) ->
+    EvList =
+     case is_list(Ev) andalso length(Ev) =:= ?POW_EV_SIZE of
+          true ->
+              Ev;
+          false ->
```

Comment:
This is like open ended. It is ok for now, we may need to make it more specific in future.